<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FightWindow</class>
 <widget class="QWidget" name="FightWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>495</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QProgressBar" name="hpBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="format">
    <string>%p</string>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>31</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>HP</string>
   </property>
  </widget>
  <widget class="QLabel" name="messageBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>320</y>
     <width>391</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>100</y>
     <width>51</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Energy</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="energyBar">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>130</y>
     <width>118</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>100</number>
   </property>
   <property name="format">
    <string>%p</string>
   </property>
  </widget>
  <widget class="QPushButton" name="okButton">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>320</x>
     <y>450</y>
     <width>93</width>
     <height>28</height>
    </rect>
   </property>
   <property name="text">
    <string>Ok</string>
   </property>
  </widget>
  <widget class="QLabel" name="monsterDisplay">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>30</y>
     <width>251</width>
     <height>271</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::WinPanel</enum>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>400</y>
     <width>395</width>
     <height>30</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QPushButton" name="attackButton">
      <property name="text">
       <string>attack</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="specialButton">
      <property name="text">
       <string>special</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="defendButton">
      <property name="text">
       <string>defend</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="waitButton">
      <property name="text">
       <string>wait</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
